wasm/wasm_source/Cargo.lock

```diff
@@ -57,6 +57,7 @@ dependencies = [
  "hex",
  "ibc",
  "ibc-proto",
+ "ics23",
```

Comment:
unrelated?

Reply:
seems related

Reply:
I think this was perhaps missing on master